app/main.py

```diff
@@ -1,8 +1,28 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        Person.people[self.name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_instances = [
+        Person(person["name"],
+               person["age"])
+        for person in people
+    ]
+
+    for person_instance in person_instances:
+        person_data = next(
+            (person for person in people if person["name"]
+             == person_instance.name),
+            None)
+
+        if person_data.get("wife"):
+            person_instance.wife = Person.people.get(person_data["wife"])
+        if person_data.get("husband"):
+            person_instance.husband = Person.people.get(person_data["husband"])
```

Comment:
The code checks for 'wife' and 'husband' keys in `person_data` and attempts to set `wife` and `husband` attributes on `person_instance`. However, if these keys are not present, `person_data.get()` will return `None`, which is handled correctly. Ensure that the logic for setting these attributes is correct and that the `Person` class is designed to handle these relationships.